### DISCUSSION TITLE
[Projects Beta] Allow view grouped by assignee

### DISCUSSION THREAD

**Discussion by sylviepaquette** [community/community#7279] [Projects and Issues] [answered]
This would be helpful for visualizing workload distribution. Currently I have created a view for this and I manually reorder the issues to group by assignee.

**Comment by azenMatt** [comment, marked as answer]
Hi @sylviepaquette - we just released this 😁  

https://github.blog/changelog/2021-12-16-the-new-github-issues-12-16-update/

Check it out and let me know what you think!
